src/main/java/uk/gov/pay/adminusers/service/ServiceUserRemover.java

```diff
@@ -37,6 +37,15 @@ public void remove(String userExternalId, String removerExternalId, String servi
 
         serviceRoleDao.remove(userServiceRoleToRemove);
     }
+    
+    public void remove(String userExternalId, String serviceExternalId) {
```

Comment:
This overloaded pattern is quite neat but as the other remove method is also just a number of strings (3 rather than 2) should this method be named something to show that it's explicitly removing without validation checks? If these functionalities were to be extended I could see accidentally calling the wrong remove by passing in the wrong number of strings. Maybe something like `removeWithoutAdminCheck` ?